tock/hours/forms.py

```diff
@@ -261,6 +261,17 @@ def clean(self):
 
         return getattr(self, 'cleaned_data', None)
 
+    def save(self, commit=True):
+        """
+        Save with deferred constraints
+        Allowing users to swap project IDs between TimeCardObjects
+        """
```

Comment:
Since this gets a bit down into the gears of things, can we either write up more on what we're doing or is there a link we can offer that gets into the details?

Reply:
Wrote this up more comprehensively in #887

Reply:
Thats great, but I'm thinking of future person reading forms.py who's confused by what this is and why we did it. I want to leave them a bit more context here (or link to more context)

Reply:
Gotcha! Added details clarifying the custom save behavior here.

Reply:
This looks great. Thank you!